docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -1466,19 +1485,20 @@ Additionally, `OidcWiremockTestResource` set token issuer and audience to `https
 [[integration-testing-keycloak-devservices]]
 === Dev services for Keycloak
 
-Using xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak] is recommended for the integration testing against Keycloak.
-`Dev Services for Keycloak` will launch and initialize a test container: it will create a `quarkus` realm, a `quarkus-app` client (`secret` secret) and add `alice` (`admin` and `user` roles) and `bob` (`user` role) users, where all of these properties can be customized.
+Use xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak] for integration testing against Keycloak.
```

Comment:
It appears to me that this sentence does not have a  message that this is a recommended way of testing. propose to revert